modular_skyrat/modules/pod_locking/pod_locking.dm

```diff
@@ -0,0 +1,22 @@
+/obj/docking_port/mobile/pod/register()
+	. = ..()
+	if(CONFIG_GET(number/minimum_alert_for_pods) != 0)
+		launch_status = NOLAUNCH
+		RegisterSignal(SSsecurity_level, COMSIG_SECURITY_LEVEL_CHANGED, PROC_REF(check_for_evac))
+
+/obj/docking_port/mobile/pod/proc/check_for_evac(datum/source, new_level)
```

Comment:
code doc pls

Reply:
done. Not sure how this links work though